[NO-ISSUE] updated all packages, fixed configs, updated documentation (#575)
leaderboardsgg/leaderboard-site: FILESTRUCTURE.md

> # Vue File Structure

Reviewer: Could simply say at the start that we follow Vue's default file structure.

Author: changing this to Nuxt, and since Nuxt file structure is a bit odd and not the norm for many frameworks like base Vue, React, Angular, etc I think it's fine to have it and keep it as it's own MD that if someone wants to read they can

## Special Directories

You can create the following extra directories, some of which have special behaviors. Only `pages` is required; you can delete them if you don't want to use their functionality.

### `assets`

The assets directory contains your uncompiled assets such as Stylus or Sass files, images, or fonts.

More information about the usage of this directory in [the documentation](https://nuxtjs.org/docs/2.x/directory-structure/assets).

### `components`

The components directory contains your Vue.js components. Components make up the different parts of your page and can be reused and imported into your pages, layouts and even other components.

More information about the usage of this directory in [the documentation](https://nuxtjs.org/docs/2.x/directory-structure/components).

### `composables`

The composables directory contains your [Vue.js composables](https://vuejs.org/guide/reusability/composables.html).

More information about the usage of this directory in [the documentation](https://v3.nuxtjs.org/guide/directory-structure/composables).

### `i18n`

> `i18n` (short for internationalizaion), is where we'll be storing our locales for translating this site into different languages.

Reviewer: Typo, and we may as well use present tense, too. Pretty sure we'll miss updating this page/not care to do so later on.

```suggestion
`i18n` (short for internationalization), is where we store our locales for translating this site into different languages.
```
